release notes cli v1.18.2 (#2679)
astronomer/docs: astro/cli/release-notes.md

This document provides a summary of all changes made to the [Astro CLI](cli/overview.md). For general product release notes, go to [Astro Release Notes](release-notes.md). If you have any questions or a bug to report, contact [Astronomer support](https://cloud.astronomer.io/support).

## Astro CLI 1.18.2

Release date: August 10, 2023

### Bug fix

> - Fixed an issue where the CLI was creating hosted Deployments where DAG-Deploy was turned off by default.

Reviewer: ```suggestion
- Fixed an issue where running `astro deployment create` on Astro Hosted would create Deployments where DAG-only deploys were turned off by default.
```
